nixos/modules/services/web-servers/unit/default.nix

```diff
@@ -89,43 +89,49 @@ in {
 
     systemd.services.unit = {
       description = "Unit App Server";
-      after = [ "network.target" ];
+      after = [ "network-online.target" ];
```

Comment:
What is this trying to fix? Usually, `network.target` should be sufficient.

Reply:
There were cases when the service did not strated after a reboot.

Reply:
`network.target` should be fine in most cases, and if services are misbehaving, I'd rather see them fixed instead of using `network-online.target` here - See https://bugzilla.redhat.com/show_bug.cgi?id=1119787#c0 for an explanation.

Reply:
Fixed.